catalog/spatial/registry/registry-admin-modules/registry-admin-local-ui/src/main/webapp/js/model/Field.js

```diff
@@ -111,7 +111,7 @@ define([
                     return;
                 }
             } else {
-                if (this.isEmpty() && !this.get('regex') && !this.get('required')) {
+                if (!this.get('value') && !this.get('regex') && !this.get('required')) {
```

Comment:
`!this.get('value')` returns `false` after `this.set('value', []);`, so the early return is no longer called. @clockard, @mcalcote, and I found a few little issues with the error messages and have written up a ticket to refactor this code: [DDF-2758](https://codice.atlassian.net/browse/DDF-2758). This PR should serve as a quick fix until that ticket is completed.